ruby/ruby/Ice.rb

```diff
@@ -233,20 +233,20 @@ def FormatType.each(&block)
 #
 require 'Ice/BuiltinSequences.rb'
 require 'Ice/Context.rb'
-require 'IceLocal/Current.rb'
-require 'IceLocal/Communicator.rb'
-require 'IceLocal/EndpointSelectionType.rb'
+require_relative 'IceLocal/Current.rb'
```

Comment:
We should also use require_relative for the Ice/ generated files. For a follow-up PR.